metadata.json

```diff
@@ -22,43 +29,23 @@
       "version_requirement": ">= 2.0.0 < 4.0.0"
     }
   ],
-  "description": "Install/configure MySQL with Galera/XtraDB cluster support",
-  "license": "BSD-2-Clause",
-  "name": "fraenki-galera",
   "operatingsystem_support": [
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "8",
-        "9"
+        "8"
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "16.04",
-        "18.04"
+        "16.04"
       ]
     },
     {
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "6",
-        "7",
```

Comment:
Why was version 6 and 7 removed?

Reply:
Same as above

Reply:
Hm i think this was ok before there was a leftover 7 a couple of lines later in the diff. Anyway I removed Scientific and Oracle completely and it is much more clean now.